manager/integration/tests/test_recurring_job.py

```diff
@@ -227,8 +229,14 @@ def recurring_job_labels_test(client, labels, volume_name, size=SIZE, base_image
     volume.attach(hostId=host_id)
     volume = common.wait_for_volume_healthy(client, volume_name)
 
-    # 5 minutes
-    time.sleep(300)
+    # 1 minutes 15s
+    time.sleep(75)
+    labels["we-added-this-label"] = "definitely"
+    jobs[0]["labels"] = labels
+    volume = volume.recurringUpdate(jobs=jobs)
+
+    # 2 minutes 15s
+    time.sleep(135)
```

Comment:
Did we verify the existence of the label after updated recurring jobs?

Reply:
Yes label verification is done below the for loop, where the backups.labels are compared against the labels object